src/pages/Environments.js

```diff
@@ -132,82 +179,173 @@ const Environments = () => {
+      onClick: () => setDeleteRuntimeId(id)
+    }, [icon('trash')])
+  }
+
+  const renderErrorApps = app => {
+    return h(Fragment, [
+      app.status[0] + app.status.substring(1).toLowerCase(),
+      app.status === 'ERROR' && h(Clickable, {
+        tooltip: 'View error',
+        'aria-label': 'View error',
+        onClick: () => setErrorAppId(app.appName)
+      }, [icon('warning-standard', { style: { marginLeft: '0.25rem', color: colors.danger() } })])
+    ])
+  }
+
+  const renderErrorRuntimes = runtime => {
+    return h(Fragment, [
+      runtime.status,
+      runtime.status === 'Error' && h(Clickable, {
+        tooltip: 'View error',
+        'aria-label': 'View error',
+        onClick: () => setErrorRuntimeId(runtime.id)
+      }, [icon('warning-standard', { style: { marginLeft: '0.25rem', color: colors.danger() } })])
+    ])
+  }
 
   return h(FooterWrapper, [
     h(TopBar, { title: 'Cloud Environments' }),
     div({ role: 'main', style: { padding: '1rem', flexGrow: 1 } }, [
       div({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase', marginBottom: '1rem' } }, ['Your cloud environments']),
       runtimes && h(SimpleFlexTable, {
-        rowCount: filteredRuntimes.length,
+        rowCount: filteredCloudEnvironments.length,
         columns: [
           {
             headerRenderer: () => h(Sortable, { sort, field: 'project', onSort: setSort }, ['Billing project']),
             cellRenderer: ({ rowIndex }) => {
-              const runtime = filteredRuntimes[rowIndex]
-              const inactive = !_.includes(runtime.status, ['Deleting', 'Error']) &&
-                currentRuntime(runtimesByProject[runtime.googleProject]) !== runtime
-              return h(Fragment, [
-                runtime.googleProject,
-                inactive && h(TooltipTrigger, {
-                  content: 'This billing project has multiple active cloud environments. Only the most recently created one will be used.'
-                }, [icon('warning-standard', { style: { marginLeft: '0.25rem', color: colors.warning() } })])
-              ])
+              const cloudEnvironment = filteredCloudEnvironments[rowIndex]
+              return cloudEnvironment.appName ? renderBillingProjectApp(cloudEnvironment) : renderBillingProjectRuntime(cloudEnvironment)
+            }
+          },
+          {
+            size: { basis: 90, grow: 0 },
+            headerRenderer: () => h(Sortable, { sort, field: 'created', onSort: setSort }, ['Type']),
+            cellRenderer: ({ rowIndex }) => {
+              const cloudEnvironment = filteredCloudEnvironments[rowIndex]
+              // TODO: update return logic once we support more app types (will need a backend change to return appType in list apps endpoint as well)
+              return cloudEnvironment.appName ? 'Galaxy' : (cloudEnvironment.runtimeConfig.cloudService === 'DATAPROC' ? 'Dataproc' : cloudEnvironment.runtimeConfig.cloudService)
```

Comment:
hmm... what exactly is this trying to do, check the appType? I see you have a TODO to update it once we return that and its a bit of scope creep (can't believe we aren't returning this!), but right now this is a nested ternary where the first condition is always false I'm fairly sure. App name is auto-generated in the UI, like GCE runtimes. I added a print with an app I created printing the value to confirm. 

![image](https://user-images.githubusercontent.com/6465084/117037558-9c48e800-acd4-11eb-8b89-d432e8ba5103.png)